.github/workflows/coverage.yml

```diff
@@ -1,3 +1,4 @@
+# TODO: Update this workflow API coverage tool is ready to handle yaml files
```

Comment:
I think you want  https://github.com/opensearch-project/opensearch-api-specification/pull/195